eng/pipelines/daily-release.yml

```diff
@@ -0,0 +1,35 @@
+
+trigger:
+  branches:
+    include:
+      - main
+
+jobs:
+  - job: build_and_commit
+    displayName: Build and Commit
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    steps:
+      - checkout: self
+
+      - script: |
+          npm install
+          npm run build
+        workingDirectory: website/
+        displayName: 'Build Website'
+
+      - script: |
+          git config --global user.email "azuresdk@microsoft.com"
```

Comment:
You should pass these as part of the commit command similar to https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/git-branch-push.ps1#L94 so you don't mess with the global state on the machine.